static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -75,13 +83,42 @@ function ConsoleMessage({breadcrumb, isLast}: ConsoleMessageProps) {
       <Icon isLast={isLast} level={breadcrumb.level}>
         {ICONS[breadcrumb.level]}
       </Icon>
-      <Message isLast={isLast} level={breadcrumb.level}>
-        <MessageFormatter breadcrumb={breadcrumb} />{' '}
-      </Message>
+      <MessageWrapper>
```

Comment:
Instead of adding a wrapper, we should change the grid template in the parent element.